client/modules/settings/components/ReceiptFields.js

```diff
@@ -0,0 +1,96 @@
+import React from 'react'
+import {Clearfix, Col, Row, Button} from 'react-bootstrap'
+
+import {Box, BoxBody, BoxHeader} from '../../../components/box'
+import {RFFieldGroup} from '../../../components/form'
+
+import { FieldArray } from 'redux-form'
+
+const renderField = ({ input, label, type, meta: { touched, error } }) => (
```

Comment:
I don't think this is needed, RFFieldGroup already renders an input so you can just remove this function and the component props from the field groups below and it still works.